### DISCUSSION TITLE
Breakup kick?

### DISCUSSION THREAD

**Discussion by omegahim** [YimMenu/YimMenu#1422] [Q&A] [answered]
The breakup kick option is missing. What happened to it?

**Comment by Yimura** [comment]
Has been patched by R*, they finally did something :)

**Reply by omegahim** [reply]
Are there any kicks that might still work against griefing modders?

**Reply by Yimura** [reply]
Try desync kick, I heard it’s the only unblock able one except if they’re the host.

**Reply by omegahim** [reply]
Desync kick doesn't work sadly, tried it before the update too, nothing was happening when I clicked it

**Reply by maybegreat48** [reply]
You have to wait around 10 seconds for the player to be kicked, assuming that the host doesn't block it for the entire session

**Reply by omegahim** [reply]
That's what I was doing. I was the host too. No effect.

**Reply by maybegreat48** [reply]
It's bugged when you are the host. I will add a host kick in my next PR

**Reply by omegahim** [reply]
PR? What does it stand for?
Sorry, not familiar with all these terms

**Reply by Yimura** [reply]
PR = pull request

**Reply by omegahim** [reply]
Ah, thx

**Comment by Tayipc38** [comment, marked as answer]
The breakup is patched for non-host participants, we have added a Host-kick option that works regardless once you are host.
